benchmark/src/main.rs

```diff
@@ -4,6 +4,14 @@ extern crate clap;
 extern crate evmap;
 extern crate rand;
 extern crate zipf;
+extern crate parking_lot;
+
+#[cfg(all(not(target_env = "msvc"), feature = "nightly"))]
+extern crate jemallocator;
+
+#[cfg(all(not(target_env = "msvc"), feature = "nightly"))]
+#[global_allocator]
+static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
```

Comment:
This only improves the performance of the benchmarker, which I don't think should be a goal, so I'd cut this.

Reply:
I added this because there were massive performance regressions on nightly that could be explained by the switch from jemalloc to the system allocator. Oddly enough, it also affected the MSVC target, which I guess doesn’t have a fix like this.

Reply:
Huh, how weird. Well, I still don't think it should be merged into `evmap` proper as I assume the underlying `nightly` issue will be fixed?

Reply:
I don’t know what else would cause it. I should note it affected the benchmark before I added anything, too, so whatever it is isn’t too relevant to this PR, but I included that to reduce future headaches. I can certainly remove it if you’d rather the benchmark stay agnostic to such things.

Reply:
Yeah, I think I would. In particular, I'd like the benchmark to be somewhat representative of what people would likely get if they dropped `evmap` into their existing implementation, which doesn't quite hold if we require them to change the system allocator. Even if the system allocator is seemingly completely broken on some systems.